refinery/ui/source/js/home/services/home-config-service.js

```diff
@@ -40,7 +40,7 @@
      * @memberOf refineryHome.getConfigs
     **/
     function getConfigs () {
-      var configs = siteProfileService.query();
+      var configs = siteProfileService.query({ current_site: 'true' });
```

Comment:
For params being sent, it appears we typically sent it to default to django's 'True'.